app/src/main/java/com/example/concordia_campus_guide/Models/Buildings.java

```diff
@@ -18,6 +18,14 @@ public Buildings(){
         return Buildings;
     }
 
+    public List<Place> getPlaces(){
+        List<Place> toReturn = new ArrayList<Place>();
```

Comment:
Here too, a more descriptive name would be "buildings".

Reply:
sounds good